typescript/api/controllers/webservice/RecordController.ts

```diff
@@ -1080,6 +1080,19 @@ export module Controllers {
       return this.apiFailWrapper(req, res, 400, null, null, "Invalid request");
     }
 
+    private isMetadataEqual(meta1:any, meta2:any): boolean {
+
+      let keys = _.keys(meta1);
+
+      for(let key of keys) {
+        if(_.get(meta1,key,'') != _.get(meta2,key,'')) {
```

Comment:
This may be faster that using _.isEqual,  just not sure whether we'll get false positives with this approach?
I guess the alternative would be 
`if(!_.isEqual(meta1?.key, meta2?.key)) {`